src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -5354,6 +5379,14 @@
           PowerSource:
               BatChargeLevel:
                   OK: Ok
+          WiFiNetworkDiagnostics:
+              WiFiVersionType:
+                  A: 80211a
+                  B: 80211b
+                  G: 80211g
+                  N: 80211n
+                  Ac: 80211ac
+                  Ax: 80211ax
```

Comment:
Stupid question but I'm wondering why 80211 is here when it is being removed everywhere else?

Reply:
Copy/pasting my comment from the other instance:

If you look at line 5222, you'll see this is in the "deprecated" section, and so old values are put in here so auto-generated things can properly map old to new, for deprecation macros.

Reply:
This is actually the "renames" section.  It's going here so the codegen knows that "A" used to be called "80211a" and can output the right old-and-new definitions and the right deprecation messages.